README.md

```diff
@@ -12,7 +12,19 @@ Performs a migration from CFME 5.7 to 5.8 utilizing steps from [Migrating to Red
 * Disks have already been resized.
 * Environments are not utilizing database replication.
 
+#### Additionaly required playbooks
+* cfme-repositories.yml
+
 #### Required groups
 * cfme
 * cfme-appliancees
 * cfme-databases
+
+### cfme-repositories.yml
+Disables all currently configured repositories and enables the required playbooks from migrate-5.7-to-5.8.yml.
+
+### configure_noop_vmware_rhel_vms.yml
+Configures noop for the queue scheduler for RHEL VMs on VMware infrastructure per [How to use the Noop IO Scheduler](https://access.redhat.com/solutions/109223).
```

Comment:
Since this isn't specific to CFME should this playbook live in https://github.com/rhtconsulting/ansible-cloud_utils instead?